web/src/user_groups.ts

```diff
@@ -430,7 +430,9 @@ export function is_user_in_setting_group(
 
 function get_display_name_for_system_group_option(setting_name: string, name: string): string {
     // We use a special label for the "Nobody" system group for clarity.
-    if (setting_name === "direct_message_permission_group" && name === "Nobody") {
+    if (setting_name === "can_resolve_topics_group" && name === "Nobody") {
+        return $t({defaultMessage: "Disable resolving topics"});
+    } else if (setting_name === "direct_message_permission_group" && name === "Nobody") {
```

Comment:
I think this function is no longer needed since we use the pills UI for both of these settings. This function can be removed in a preparatory commit.